packages/nodes-base/tslint.json

```diff
@@ -61,12 +65,15 @@
 		"no-default-export": true,
 		"no-duplicate-variable": true,
 		"no-inferrable-types": true,
-		"ordered-imports": [true, {
-			"import-sources-order": "any",
-			"named-imports-order": "case-insensitive"
-		  }],
-		"no-namespace": [
+		"ordered-imports": [
 			true,
+			{
+				"import-sources-order": "any",
+				"named-imports-order": "case-insensitive"
+			}
+		],
+		"no-namespace": [
```

Comment:
I checked the docs about the `no-namespace` rule and it's an outdated mode of organizing the code, as can be seen here: https://palantir.github.io/tslint/rules/no-namespace/

Is there an alternative to achieve the same results?

Reply:
I think that opinion itself is outdated, it's from 2018
©2018 Palantir Technologies under [Apache 2.0](https://github.com/palantir/tslint/blob/master/LICENSE)


The official TS docs list it as a viable solution to separating code.
https://www.typescriptlang.org/docs/handbook/namespaces.html#namespacing